src/mlpack/methods/ann/layer/lstm_impl.hpp

```diff
@@ -159,10 +159,24 @@ void LSTM<InputDataType, OutputDataType>::Reset()
       offset, outSize, 1, false, false);
 }
 
+// Forward when cellState is not needed
```

Comment:
Full stop at the end of the sentence please.

Reply:
Done